### DISCUSSION TITLE
Filter status code FAILED

### DISCUSSION THREAD

**Discussion by 0xCGonzalo** [projectdiscovery/httpx#844] [Q&A] [answered]
Is it possible to filter by httpx response code (e.g. when FAILED returns that is not displayed in the output)?

![image](https://user-images.githubusercontent.com/43796175/198912119-20cf5695-8894-45b2-8011-9c912303752b.png)

**Comment by ehsandeep** [comment, marked as answer]
@0xCGonzalo if you can stop using `-ms` flag if you don't wish to see the failed target in the result?
